### PR TITLE
Add Zomp EF Core Extensions

### PR DESCRIPTION
**Pull request by virzak**
As per this suggestion: https://github.com/dotnet/efcore/issues/12747#issuecomment-1192517531

### CODE REVIEW

**Comment by ErikEJ** [comment]
LGTM!

**Comment by virzak** [comment]
Not sure how to best add multiple nuget packages to that template though.

File: entity-framework/core/extensions/index.md [hunk 1]
```diff
@@ -345,6 +345,12 @@ Cloning entities using EF Core configuration. You can use the `Include` method t
 
 [GitHub repository](https://github.com/HenkKin/EntityCloner.Microsoft.EntityFrameworkCore) | [NuGet](https://www.nuget.org/packages/EntityCloner.Microsoft.EntityFrameworkCore)
 
+### Zomp EF Core Extensions
+
+Provides window (analytics) functions and binary functions for EF Core. Providers: SQL Server, SQLite, Postgres. For EF Core: 6, 7.
```
Comment (roji) [inline entity-framework/core/extensions/index.md:350]:
```suggestion
Provides window (analytics) functions and binary functions for EF Core. Providers: SQL Server, SQLite, PostgreSQL. For EF Core: 6, 7.
```

**Review by roji — APPROVED** [review]
@virzak thanks.

**Pull request merged**